pyccel/codegen/printing/fcode.py

```diff
@@ -1236,7 +1236,9 @@ def _print_Allocate(self, expr):
             code += 'end if\n'
 
         return code
-
+#-----------------------------------------------------------------------------
+    def _print_Deallocate(self, expr):
+        return ''
```

Comment:
If you want to check that you haven't missed any or deallocated twice you could implement this method:
```
return 'deallocate({})'.format(self._print(expr.variable))
```
Fortran makes lots of errors if this is not done properly ^^'

It is not necessary though (deallocate(null) doesn't work in fortran)